ttnn/cpp/ttnn/tensor/types.hpp

```diff
@@ -34,7 +34,12 @@ SimpleShape is a temporary measure aimed at making a clear distinction between S
 **/
 class SimpleShape {
 public:
-    template <typename T>
+    SimpleShape() = default;
```

Comment:
Are you sure we need a default constructor for SimpleShape?

Reply:
I see that this constructor make sense for Alignment, but that shows another reason to separate them